nservicebus/bridge/configuration_failed-messages_bridge_[2,).partial.md

```diff
@@ -0,0 +1,16 @@
+### Handling of failed messages
+
+By default, when the bridge transfers a message to the ServiceControl transport it will not attempt to translate the [`NServiceBus.ReplyToAddress`](/nservicebus/messaging/headers.md#messaging-interaction-headers-nservicebus-replytoaddress) message header.
+This means that endpoints only need to be registered with the bridge if they are directly involved with messages transferred between transports.
+The drawback is that if an endpoint is moved to a different transport, then any failed messages from that endpoint cannot be retried since the value in the `NServiceBus.ReplyToAddress` header no longer exists.
+
+This behavior can be changed by configuring the bridge to translate the `NServiceBus.ReplyToAddress` message header for failed messages.
+
+snippet: translate-reply-to-address-for-failed-messages
+
+> [!WARNING]
+> Since any endpoint can generate a failed message, enabling translation of the `NServiceBus.ReplyToAddress` header requires that all endpoints are registered with the bridge.
```

Comment:
```suggestion
> Since any endpoint can generate a failed message, enabling translation of the `NServiceBus.ReplyToAddress` header requires that all endpoints participating in the 'conversation' (either as receivers of the message or part of the `ReplyTo` chain) are registered with the bridge.
```

Reply:
As discussed on call, this change isn't necessary since all endpoints that generate errors (which need to be bridged to SC) will have a ReplyTo header that needs translation